[bitnami/kafka] fix kafka_jaas config filepath in KAFKA_OPTS

**Which chart**:
bitnami/kafka

**Describe the bug**
Incorrect filepath specified for `java.security.auth.login.config` in `KAFKA_OPTS`.

Currently, the env variable `KAFKA_OPTS` has the below value specified in the [statefulset.yaml](https://github.com/bitnami/charts/blob/master/bitnami/kafka/templates/statefulset.yaml#L195).

```
            - name: KAFKA_OPTS
              value: "-Djava.security.auth.login.config=/opt/bitnami/kafka/conf/kafka_jaas.conf"
```

The `kafka_jaas.conf` file is however generated and stored at `/opt/bitnami/kafka/config/kafka_jaas.conf` and not at `/opt/bitnami/kafka/conf/kafka_jaas.conf`, as per latest code in [bitnami/bitnami-docker-kafka](https://github.com/bitnami/bitnami-docker-kafka/blob/master/2/debian-10/rootfs/opt/bitnami/scripts/libkafka.sh#L314).

Kafka CLI operations fail to find jaas config file for authentication using the above configuration for `KAFKA_OPTS`. Request to update the path in `KAFKA_OPTS` env var with the latest path.


**To Reproduce**
Steps to reproduce the behavior:
1. Setup kafka and zookeeper with security enabled.
2. Execute any kafka CLI operation (eg. list topics)

**Expected behavior**
Kafka CLI operation to find jaas config file at `/opt/bitnami/kafka/config/kafka_jaas.conf` and execute successfully.

**Actual behaviour**
```
I have no name!@kafka-0:/opt/bitnami/kafka$ bin/kafka-topics.sh --zookeeper kafka-zookeeper:2181 --list
Exception in thread "main" org.apache.kafka.common.KafkaException: Exception while loading Zookeeper JAAS login context [java.security.auth.login.config=/opt/bitnami/kafka/conf/kafka_jaas.conf, zookeeper.sasl.client=default:true, zookeeper.sasl.clientconfig=default:Client]
        at org.apache.kafka.common.security.JaasUtils.isZkSaslEnabled(JaasUtils.java:67)
        at kafka.admin.TopicCommand$ZookeeperTopicService$.apply(TopicCommand.scala:360)
        at kafka.admin.TopicCommand$.main(TopicCommand.scala:55)
        at kafka.admin.TopicCommand.main(TopicCommand.scala)
Caused by: java.lang.SecurityException: java.io.IOException: /opt/bitnami/kafka/conf/kafka_jaas.conf (No such file or directory)
        at java.base/sun.security.provider.ConfigFile$Spi.<init>(ConfigFile.java:137)
        at java.base/sun.security.provider.ConfigFile.<init>(ConfigFile.java:102)
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
        at java.base/java.lang.Class.newInstance(Class.java:584)
        at java.base/javax.security.auth.login.Configuration$2.run(Configuration.java:255)
        at java.base/javax.security.auth.login.Configuration$2.run(Configuration.java:246)
        at java.base/java.security.AccessController.doPrivileged(Native Method)
        at java.base/javax.security.auth.login.Configuration.getConfiguration(Configuration.java:245)
        at org.apache.kafka.common.security.JaasUtils.isZkSaslEnabled(JaasUtils.java:63)
        ... 3 more
Caused by: java.io.IOException: /opt/bitnami/kafka/conf/kafka_jaas.conf (No such file or directory)
        at java.base/sun.security.provider.ConfigFile$Spi.ioException(ConfigFile.java:665)
        at java.base/sun.security.provider.ConfigFile$Spi.init(ConfigFile.java:262)
        at java.base/sun.security.provider.ConfigFile$Spi.<init>(ConfigFile.java:135)
        ... 14 more
```

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
Client: &version.Version{SemVer:"v2.16.9", GitCommit:"8ad7037828e5a0fca1009dabe290130da6368e39", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.16.9", GitCommit:"8ad7037828e5a0fca1009dabe290130da6368e39", GitTreeState:"clean"}
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"16+", GitVersion:"v1.16.6-beta.0", GitCommit:"e7f962ba86f4ce7033828210ca3556393c377bcc", GitTreeState:"clean", BuildDate:"2020-01-15T08:26:26Z", GoVersion:"go1.13.5", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"16+", GitVersion:"v1.16.6-beta.0", GitCommit:"e7f962ba86f4ce7033828210ca3556393c377bcc", GitTreeState:"clean", BuildDate:"2020-01-15T08:18:29Z", GoVersion:"go1.13.5", Compiler:"gc", Platform:"linux/amd64"}
```

--- a/bitnami/kafka/Chart.yaml
+++ b/bitnami/kafka/Chart.yaml
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kafka
-version: 11.8.1
+version: 11.8.2
 appVersion: 2.6.0
 description: Apache Kafka is a distributed streaming platform.
 keywords:

--- a/bitnami/kafka/templates/statefulset.yaml
+++ b/bitnami/kafka/templates/statefulset.yaml
@@ -192,7 +192,7 @@ spec:
               value: {{ ternary "yes" "no" (or .Values.auth.enabled .Values.allowPlaintextListener) | quote }}
             {{- if or (include "kafka.client.saslAuthentication" .) (include "kafka.interBroker.saslAuthentication" .) }}
             - name: KAFKA_OPTS
-              value: "-Djava.security.auth.login.config=/opt/bitnami/kafka/conf/kafka_jaas.conf"
+              value: "-Djava.security.auth.login.config=/opt/bitnami/kafka/config/kafka_jaas.conf"
             {{- if (include "kafka.client.saslAuthentication" .) }}
             - name: KAFKA_CLIENT_USERS
               value: {{ join "," .Values.auth.jaas.clientUsers | quote }}

